src/analysis/AppLauncher.ts

```diff
@@ -359,10 +420,13 @@ const ApplicationLauncher = _.flow(
                   [_.includes(runtimeStatus, usableStatuses), () => 'Almost ready...'],
                   [
                     runtimeStatus === 'Stopping',
-                    () => 'Cloud environment is stopping, which takes ~4 minutes. You can restart it after it finishes.',
+                    () =>
+                      'Cloud environment is stopping, which takes ~4 minutes. You can restart it after it finishes.',
+                  ],
+                  [
+                    runtimeStatus === 'Stopped',
+                    () => 'Cloud environment is stopped. Start it to edit your notebook or use the terminal.',
                   ],
-                  [runtimeStatus === 'Stopped', () => 'Cloud environment is stopped. Start it to edit your notebook or use the terminal.'],
-                  [runtimeStatus === 'LeoReconfiguring', () => 'Cloud environment is updating, please wait.'],
```

Comment:
why don't we need this updating message anymore?

Reply:
this isn't actually a valid status leo returns, `Updating` is a usable status, this was some wonky UI sanitization status that typing has stamped out.